### PR TITLE
Added prelease warning

### PR DESCRIPTION
**Pull request by kuzmatech**
Added a warning to avoid user issues such as the complaints in #107 and similar issues.

### CODE REVIEW

File: docs/content/en/setup.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ Add `@nuxtjs/google-fonts` dependency to your project:
   </code-block>
 </code-group>
 
+>**Warning**
+When using Nuxt 3 and/or Nuxt Bridge ensure you're using the prelease version `@nuxtjs/google-fonts@prerelease`.
```
Comment (hartmut-co-uk) [inline docs/content/en/setup.md:32]:
```suggestion
When using Nuxt 3 and/or Nuxt Bridge ensure you're using the pre-release version `@nuxtjs/google-fonts@prerelease`.
```

File: docs/content/en/setup.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Add `@nuxtjs/google-fonts` dependency to your project:
 </code-group>
 
 >**Warning**
-When using Nuxt 3 and/or Nuxt Bridge ensure you're using the prelease version `@nuxtjs/google-fonts@prerelease`.
+When using Nuxt 3 and/or Nuxt Bridge ensure you're using the prerelease version `@nuxtjs/google-fonts@prerelease`.
 
 Then, add `@nuxtjs/google-fonts` to the `buildModules` section of `nuxt.config.js`:
```
Comment (hartmut-co-uk) [inline docs/content/en/setup.md:34]: Nuxt3 does no longer differentiate _modules_:

```suggestion
Then, add `@nuxtjs/google-fonts` to the `modules` section of `nuxt.config.js`:
```

Comment (hartmut-co-uk) [reply docs/content/en/setup.md:34]: hmm I can't do it properly as a suggestion, I think this should also go under 'nuxt3 warning' section.

**Review by iamandrewluca — APPROVED** [review]

**Review by manchenkoff — APPROVED** [review]

**Comment by ricardogobbosouza** [comment]
No need any more...

Released [v3](https://github.com/nuxt-community/google-fonts-module/releases/tag/v3.0.0)

**Pull request closed without merging**